scripts/generateTxTags.js

```diff
@@ -27,7 +28,7 @@ websocket.onmessage = message => {
   const registry = new TypeRegistry();
 
   const metadata = new Metadata(registry, JSON.parse(message.data).result);
-  const modules = metadata.asLatest.modules;
+  const modules = metadata.asLatest.pallets;
```

Comment:
There are other changes we should investigate: https://polymath.atlassian.net/browse/DA-305